File: spider-demo/src/main/java/com/ztliao/spider/domain/entity/Task.java
package com.ztliao.spider.domain.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

/**
 * @author: liaozetao
 * @date: 2021/7/31 8:32 PM
 * @description:
 */
@TableName("task")
@Data
public class Task {

    @TableId(type = IdType.UUID)
    private String id;

    private String content;

    private String flag = "N";

    private String lackStocks;
}

File: spark-demo/spark-core/src/main/scala/com/ztliao/spark/core/test/Task.scala
package com.ztliao.spark.core.test

/**
 * @author: liaozetao
 * @date: 2023/8/14 13:15
 * @description:
 */
class Task extends Serializable {

  var datas: List[Int] = _

  var logic: (Int) => Int = _


}
